apps/frontend/src/renderer/components/settings/SecretsManager.tsx

```
@@ -0,0 +1,432 @@
+  SelectTrigger,
+  SelectValue,
+} from '../ui/select';
+import type { SecretGroup } from '../../../shared/types';
+import { AddSecretGroupDialog } from './AddSecretGroupDialog';
+import { AddSecretAccountDialog } from './AddSecretAccountDialog';
+
+interface SecretsManagerProps {
+  // No props needed for embedded view
+}
+
+export function SecretsManager({}: SecretsManagerProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)